On Alpine, apropos(1) is available in a separate subpackages and not
installed by default. For this reason, having this warning enabled by
default does not make much sense since we don't use mandoc.db by
default.

diff -upr a/main.c b/main.c
--- a/main.c	2021-09-19 18:21:40.000000000 +0200
+++ b/main.c	2021-09-19 19:40:28.711594083 +0200
@@ -822,8 +822,6 @@ fs_lookup(const struct manpaths *paths,
 	return globres;
 
 found:
-	warnx("outdated mandoc.db lacks %s(%s) entry, run %s %s",
-	    name, sec, BINM_MAKEWHATIS, paths->paths[ipath]);
 	if (res == NULL)
 		free(file);
 	else if (file == NULL)
